app/src/androidTest/java/com/google/jetpackcamera/NavigationTest.kt

```diff
@@ -127,7 +127,8 @@ class NavigationTest {
         // Press the device's back button
         uiDevice.pressBack()
 
-        // Assert we're on PreviewScreen by finding the capture button
+        // Assert we're on PreviewScreen by finding the capture button and not the ratio button
+        composeTestRule.onNodeWithTag(QUICK_SETTINGS_RATIO_BUTTON).assertDoesNotExist()
         composeTestRule.onNodeWithTag(CAPTURE_BUTTON).assertExists()
```

Comment:
nit: This line should be unnecessary since the capture button exists on both the quick settings and preview screen.

Reply:
> nit: This line should be unnecessary since the capture button exists on both the quick settings and preview screen.

I think alternatively you could give the flip camera button (the one that always stays on the preview screen) a tag and check for that